noresm2.1/noresm2.1_release_notes.rst

```diff
@@ -2,3 +2,96 @@
+
+CAM
+~~~
+- There is a new compile-time XML configuration variable that enables `AEROCOM` diagnostic output. Simply execute this command _before_ building the code: `./xmlchange CAM_AEROCOM=TRUE`
+- All code and diagnostic output formerly controlled by the `AEROFFL` preprocessor definition is now enabled by default whenever using `OSLO_AERO`
+- The `DIRIND` preprocessor definition has been removed and the code previously contained inside the definitions are now enabled whenever using `OSLO_AERO`
+- The preprocessor definitions and associated code for `RFMIPIRF`, `SPAERO`, and `COLTST4INTCONS` have been removed. The code previously contained inside these definitions are no longer in NorESM2.1
+- Specifying the model grid alias as f19_f19 is identical to specifying the model grid alias as f19_f19_mtn14. This would apply to creating NF compsets.
+
+BLOM
+~~~~
+- Two updated release v1.4.0 and v1.5.0 Release notes are summarized at https://github.com/NorESMhub/BLOM/discussions/319 and https://github.com/NorESMhub/BLOM/discussions/313
+- v1.5.0 contained updates for hybrid coordinates
+- v1.4.0 contained major iHAMOCC changes
+  + Major iHAMOCC code re-organization
+    + most pre-processor flags ("ifdefs") replaced by logical flags that are read in via namelist
+    + all subroutines placed in modules
+    + new module mo_param_bgc collects all model parameters and routines for initialization of model parameters
+  + Major iHAMOCC code style changes (unified indentation, lower-case keywords)
+  + Added regression testing functionality for BLOM, when run as part of NorESM
+  + New mechanisms to create namelist files when run as part of NorESM (consistent with other NorESM components), through the file namelist_definition_blom.xml
+  + New mesoscale eddy diffusivity options
+  + C-isotope code for sediment now technically supported
+  + Added support for simulating Ocean Alkalinity Enhancement
+
+CISM
+~~~~
+- The first steps to enable coupled climate-ice sheet runs (KeyCLIM) are now in place for CISM for compsets N1850frc2G, NHISTfrc2G, NSSP585frc2G, NSSP585frc2extG. These cannot be currently run out of the box since they require CISM restart files that are not distributed with this release. However, this capability will be in place in NorESM2.3 in the spring.
+
+Tested  configurations (compset / resolution combinations)
+----------------------------------------------------------
+The combinations of compset and resolution below have been run on Betzy. That is why simulations using these combinations can be created without using the `--run-unsupported` option to `create_newcase` (see introduction above).
+
+Longer tests (currently running)
```

Comment:
@DirkOlivie, are these still running? If not, we should change this.

Reply:
Sorry, my mistake. Should be corrected in the release notes on github as well. Will do that now.

Reply:
No, the simulations are finished.